README.md

```diff
@@ -93,6 +95,8 @@ $ cp Loop/RemoteSettings-template.plist Loop/RemoteSettings.plist
 | `AmplitudeAPIKey`      | Your Amplitude analytics API Key (for optional, private behavior tracking)
 | `ShareAccountName`     | Your username for Dexcom share (for backfilling glucose data)
 | `ShareAccountPassword` | Your password for Dexcom share
+| `NightscoutSiteURL`    | Your Nightscout site URL
+| `NightscoutAPISecret`  | Your Nightscout API Secret (not hashed)
```

Comment:
FYI I'm going to migrate these all eventually (rather than storing in plain text in the app bundle)
#35